### PR TITLE
Images: Update license path for all plugin images

### PR DESCRIPTION
**Pull request by chaitanya1731**
Updated license path to /install_root/licenses/ for all the plugin images except - intel-qat-plugin-kerneldrv, intel-vpu-plugin and intel-idxd-config-initcontainer images.

### CODE REVIEW

**Comment by bart0sh** [comment]
@chaitanya1731 please fix e2e-fpga build failure. It's most probably caused by your changes.

**Comment by mythi** [comment]
> @chaitanya1731 please fix e2e-fpga build failure. It's most probably caused by your changes.

@bart0sh the images fail to build. that makes me think can we set dependencies to actions? That is, run e2e only after everything in `ci.yaml` is passing

**Comment by bart0sh** [comment]
@mythi that makes sense. Thanks for the suggestion!

**Review by mythi — CHANGES_REQUESTED** [review]
Thanks, this looks good but the merge commits need to be cleaned. Also, can you update:

https://github.com/intel/intel-device-plugins-for-kubernetes/blob/2e0c45b5632fc63e16226014a293b590ba905751/README.md?plain=1#L285-L287

**Comment by chaitanya1731** [comment]
Changes made as per request

File: README.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ All of the source code required to build intel-device-plugins-for-kubernetes
 is available under Open Source licenses. The source code files identify external Go
 modules used. Binaries are distributed as container images on
 DockerHub. Those images contain license texts under
-`/usr/local/share/package-licenses` and source code under
+`/licenses` and source code under
 `/usr/local/share/package-sources`.
```
Comment (mythi) [inline README.md:287]: with `go-licences` the source code is under the same path so this sentence should just say "Those images contain license texts and source code under `/licenses`"

**Review by mythi — COMMENTED** [review]
@chaitanya1731 looks good! one minor comment about the `README` updates

**Review by bart0sh — APPROVED** [review]
/lgtm

**Review by mythi — APPROVED** [review]
Thanks!

**Pull request merged**